README.md

```diff
@@ -7,6 +7,87 @@ Example uses might include:
+often need to be dereferenced in order to be most useful.
+
+Using Marmotta's `linkeddata` backend to the LDPath-based transform module, it is possible to query these resources and follow links to arbitrary depths.
+
+For example:
+
+```
+@prefix gn : <http://www.geonames.org/ontology#>
+
+title = dcterms:title :: xsd:string ;
+subject = dcterms:subject / (skos:prefLabel | rdfs:label) :: xsd:string ;
+location = dcterms:spatial / gn:name :: xsd:string ;
+```
+
+Could produce a response such as:
+
+```
+[{
+    "title" : ["Birds of Amherst, MA"] ,
+    "subject" : ["Birds", "Birds of New England"] ,
+    "location" : ["Amherst"]
+}]
+```
+
+By default, the `linkeddata` backend will cache responses from remote repositories for 1 day. It is possible to specify particular behavior on an endpoint-by-endpoint basis by configuring the Spring-based
+transform context. For instance, adding support for a Getty endpoint could include (as part of the `./spring/transform.xml` configuration):
+
+```
+<util:list id="endpoints" value-type="org.apache.marmotta.ldclient.api.endpoint.Endpoint">
+
+  <!-- add support for Getty vocabs -->
+  <bean class="org.apache.marmotta.ldclient.endpoint.rdf.SPARQLEndpoint">
+    <constructor-arg index="0" type="java.lang.String" value="Getty Vocabs"/>
+    <constructor-arg index="1" type="java.lang.String" value="http://vocab.getty.edu/sparql"/>
+    <constructor-arg index="2" type="java.lang.String" value="^http://vocab\\.getty\\.edu/.*"/>
+  </bean>
+
+
+  <bean class="org.fcrepo.transform.ldcache.FedoraEndpoint">
+    <!-- set the base URL of the Fedora endpoint (hostname and port will suffice) -->
+    <constructor-arg type="java.lang.String" value="http://localhost:8080"/>
+    <!-- cache values for five minutes (300) seconds -->
+    <!-- set this to 0 to ensure that data is always fresh -->
+    <constructor-arg type="long" value="300"/>
+  </bean>
+</util:list>
+```
+
+If your Fedora repository has authorization enabled, you will need to provide credentails to the linked data client.
+
+```
+<!-- define the username and password for the user accessing the repo with the ldclient -->
+<bean id="credentials" class="org.apache.http.auth.UsernamePasswordCredentials">
+  <constructor-arg type="java.lang.String" value="USERNAME"/>
+  <constructor-arg type="java.lang.String" value="PASSWORD"/>
+</bean>
+
+<!-- define the host and port for the repository -->
+<bean id="authScope" class="org.apache.http.auth.AuthScope">
+  <constructor-arg type="java.lang.String" value="localhost"/>
+  <constructor-arg type="int" value="8080"/>
+</bean>
+```
+
+If you customize the spring configuration, it is generally recommended to put your customization in a separate, system directory such as `/etc/fcrepo/spring/transform.xml`.
+Then, you can add the following to your `JAVA_OPTS`: `-Dfcrepo.spring.transform.configuration=file:/etc/fcrepo/spring/transform.xml`.
```

Comment:
We will need to document this configuration element (`fcrepo.spring.transform.configuration`) here:
https://wiki.duraspace.org/display/FEDORA4x/Configuration+Options+Inventory ...possibly among other places.